tests/layer/test_ppnp.py

```diff
@@ -0,0 +1,76 @@
+from stellargraph.layer.ppnp import *
```

Comment:
See my comment for the lack of tests for edge cases for APPNP. The same applies here for PPNP.